style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  margin: 1%;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+/* Header with Logo & Navbar */
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #f6fafc;
+  padding: 1em;
+}
+
+.logo {
+  margin-right: 15px;
+}
+
+.logo img {
+  max-width: 100%;
+}
+
+.navbar {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.toggle-button {
+  color: #810e0e;
+  font-size: 36px;
+  font-weight: 300;
+}
+
+.menu {
+  display: none;
+  list-style-type: none;
+}
+
+.menu li a {
+  text-decoration: none;
+}
+
+/* Main */
+#intro {
+  background-image: url("img/Rectangle\ 6.png"), url("img/Ellipse\ 11.png"), url("img/Rectangle\ 7.png"), url("img/Rectangle\ 4.png"), url("img/Ellipse\ 3.png");
+  background-repeat: no-repeat;
+  background-size: 5%;
+  background-position: left top, left 4% top 30%, center top, right 6% top, right 4% top 27%;
+}
+
+#intro h1 {
+  color: #091e42;
+  font-weight: bold;
+}
+
+.my-description {
+  position: relative;
+}
+
+#social-media-connect {
+  color: #7f8cff;
+  text-decoration: none;
+}
+
+.social-media-icon {
+  width: 35%;
+  display: flex;
+  list-style-type: none;
+  align-items: baseline;
+  padding-bottom: 55%;
+}
+
+.social-media-icon li {
+  flex: 1;
+}
+
+#socail-media {
+  background-image: url("img/Ellipse\ 12.png"), url("img/Ellipse\ 11.png"), url("img/Ellipse\ 4.png"), url("img/Rectangle\ 4.png"), url("img/Rectangle\ 5.png"), url("img/Ellipse\ 6.png");
+  background-repeat: no-repeat;
+  background-size: 6%;
+  background-position: left 5% bottom 66%, left 19% bottom 52%, center bottom 51%, center bottom 57%, right 13% bottom 58%, right 15% bottom 54%;
+}
+
+@media (min-width: 375px) {
+  .menu:hover,
+  .toggle-button:hover + .menu {
+    display: flex;
+  }
+
+  .toggle-button:hover + .menu {
+    flex-wrap: wrap;
+    margin-right: 5%;
+  }
+
+  .toggle-button:hover .menu-icon {
+    transform: rotate(45deg);
+  }
+
+  .menu:hover li,
+  .toggle-button:hover + .menu li {
+    flex-basis: calc(99% / 3);
+    padding-right: 5%;
+  }
+}
+
+/* Media query for desktop screens */
+@media (min-width: 768px) {
```

Comment:
- [ ] Great initiative, @HossainAraf, but I would ask you to remove the media query for now as it is the part of next milestone, see? desktop version